e2e/nomostest/testing/ntb.go

```diff
@@ -168,12 +176,11 @@ func (t *FakeNTB) Skipped() bool {
 
 // Log generates the output. It's always at the same stack depth.
 func (t *FakeNTB) Log(args ...interface{}) {
-	fmt.Println(args...)
+	fmt.Println(fmt.Sprintf("[%s] ", t.name) + fmt.Sprint(args...))
```

Comment:
i think you can replace the + with a comma and remove the space in the quotes. Println will join args with spaces, right?

Reply:
Yeah I think you're right, will do

Reply:
done